.github/workflows/main.yml

```diff
@@ -82,6 +82,7 @@ jobs:
       - name: Build and Push
         uses: docker/build-push-action@v2
         with:
+          context: .
```

Comment:
See docs for [GitHub Action](https://github.com/marketplace/actions/build-and-push-docker-images#path-context). Was failing the build otherwise as there was no `.git` directory: https://github.com/mcagov/beacons-service/runs/2216687436?check_suite_focus=true